eslint.config.js

```diff
@@ -15,7 +14,8 @@ export default [
 			'packages/playground/big/src/pages/**', // lots of generated files
 			'packages/*/types/index.d.ts',
 			'packages/*/types/index.d.ts.map',
-			'packages/*/CHANGELOG.md'
+			'packages/*/CHANGELOG.md',
```

Comment:
i assume these are added to avoid the "embedded svelte" errors coming from  markdown codefence sources? Would be good to have those working

Reply:
OK. I will adjust it.

Reply:
updated: https://github.com/sveltejs/vite-plugin-svelte/pull/1095/commits/cf5db5c983a8cdd7c4d73b9c3bc075bc6bff07f8

Reply:
great!